packages/aws-cdk-lib/aws-rds/lib/instance-engine.ts

```diff
@@ -571,37 +690,85 @@ export class MysqlEngineVersion {
+   */
   public static readonly VER_5_7_17 = MysqlEngineVersion.of('5.7.17', '5.7');
-  /** Version "5.7.19". */
+  /**
+   * Version "5.7.19"
+   * @deprecated MySQL 5.7.19 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_19 = MysqlEngineVersion.of('5.7.19', '5.7');
-  /** Version "5.7.21". */
+  /**
+   * Version "5.7.21"
+   * @deprecated MySQL 5.7.21 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_21 = MysqlEngineVersion.of('5.7.21', '5.7');
-  /** Version "5.7.22". */
+  /**
+   * Version "5.7.22"
+   * @deprecated MySQL 5.7.22 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_22 = MysqlEngineVersion.of('5.7.22', '5.7');
-  /** Version "5.7.23". */
+  /**
+   * Version "5.7.23"
+   * @deprecated MySQL 5.7.23 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_23 = MysqlEngineVersion.of('5.7.23', '5.7');
-  /** Version "5.7.24". */
+  /**
+   * Version "5.7.24"
+   * @deprecated MySQL 5.7.24 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_24 = MysqlEngineVersion.of('5.7.24', '5.7');
-  /** Version "5.7.25". */
+  /**
+   * Version "5.7.25"
+   * @deprecated MySQL 5.7.25 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_25 = MysqlEngineVersion.of('5.7.25', '5.7');
-  /** Version "5.7.26". */
+  /**
+   * Version "5.7.26"
+   * @deprecated MySQL 5.7.26 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_26 = MysqlEngineVersion.of('5.7.26', '5.7');
-  /** Version "5.7.28". */
+  /**
+   * Version "5.7.28"
+   * @deprecated MySQL 5.7.28 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_28 = MysqlEngineVersion.of('5.7.28', '5.7');
-  /** Version "5.7.30". */
+  /**
+   * Version "5.7.30"
+   * @deprecated MySQL 5.7.30 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_30 = MysqlEngineVersion.of('5.7.30', '5.7');
-  /** Version "5.7.31". */
+  /**
+   * Version "5.7.31"
+   * @deprecated MySQL 5.7.31 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_31 = MysqlEngineVersion.of('5.7.31', '5.7');
-  /** Version "5.7.33". */
+  /**
+   * Version "5.7.33"
+   * @deprecated MySQL 5.7.33 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_33 = MysqlEngineVersion.of('5.7.33', '5.7');
-  /** Version "5.7.34". */
+  /**
+   * Version "5.7.34"
+   * @deprecated MySQL 5.7.34 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_34 = MysqlEngineVersion.of('5.7.34', '5.7');
-  /** Version "5.7.35". */
+  /**
+   * Version "5.7.35"
+   * @deprecated MySQL 5.7.35 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_35 = MysqlEngineVersion.of('5.7.35', '5.7');
-  /** Version "5.7.36". */
+  /**
+   * Version "5.7.36"
+   * @deprecated MySQL 5.7.36 is no longer supported by Amazon RDS.
+   */
```

Comment:
It seems like v5 is now deprecated ([docs](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html))

Reply:
Again, based on the SDK:

```sh
$ aws rds describe-db-engine-versions --engine mysql --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion | startswith("5.")) | .EngineVersion,.Status'
5.7.37
available
5.7.38
available
5.7.39
available
5.7.40
available
5.7.41
available
5.7.42
available
5.7.43
available
5.7.44
available
```